spcs-basic/app/manifest.yml

```diff
@@ -0,0 +1,31 @@
+manifest_version: 1
+
+version:
+  name: v1
+  label: Version One
+
```

Comment:
I think we should remove this; it's optional and I don't think it's a best practice.